README.md

```diff
@@ -1,137 +1,167 @@
-
-#### Override karma config
-
-You can override karma config by adding a `karmaConfig` property to your `stryker.conf.js` file.
-
-For example:
-
-```javascript
-karmaConfig: {
-    browsers: ['Chrome'],
-    frameworks: ['mocha']
-}
-```
-
-*Note*: Whichever testFramework you use should also be reflected in the `testFramework` property of stryker itself. For example: `testFramework: 'mocha'`  
-
-Not all karma config can be overriden, as Stryker requires specific functionality from the testRunner to do its magic. 
-Karma config that *cannot* be overriden:
-
-* `files`: The karma-runner will fill this based on the `files` and `filesToMutate` configuration in the `stryker-conf.js` file.
-* `coverageReporter`: This will be enabled for the initial test run, disabled for testing the mutants.
-* `autoWatch`, `singleRun`: Stryker needs full control on when to run the karma tests
-
-### Full config example
-
-```javascript
-// stryker.conf.js
-exports = function(config){
-    config.set({
-        // ...
-        testRunner: 'karma',
-        testFramework: 'jasmine',
-        karmaConfig: { // these are the defaults
-            browsers: ['PhantomJS'],
-            frameworks: ['jasmine'],
-            autoWatch: false,
-            singleRun: false
-        },
-        plugins: ['stryker-karma-runner'] // Or leave out the plugin list entirely to load all stryker-* plugins directly
-        // ...
-    });
-}
-```
-
-## Usage
-
-Use Stryker as you normally would.
-See [http://stryker-mutator.github.io](http://stryker-mutator.github.io) for more info. 
-
-### Debugging
-
-By default, karma logging will be swallowed. 
-Set log level to `trace` in `stryker.conf.js` to see all the karma output.
-
-```javascript
-// stryker.conf.js
-exports = function(config){
-    config.set({
-        // ...
-        logLevel: 'trace'
-        // ...
-    });
-}
+[![Build Status](https://travis-ci.org/stryker-mutator/stryker-karma-runner.svg?branch=master)](https://travis-ci.org/stryker-mutator/stryker-karma-runner)
+[![NPM](https://img.shields.io/npm/dm/stryker-karma-runner.svg)](https://www.npmjs.com/package/stryker-karma-runner)
+[![Node version](https://img.shields.io/node/v/stryker-karma-runner.svg)](https://img.shields.io/node/v/stryker-karma-runner.svg)
+[![Gitter](https://badges.gitter.im/stryker-mutator/stryker.svg)](https://gitter.im/stryker-mutator/stryker?utm_source=badge&utm_medium=badge&utm_campaign=pr-badge)
+
+#Stryker Karma Runner
+
+A plugin to use the karma test runner in [Stryker](https://stryker-mutator.github.io), the JavaScript mutation testing framework
+
+## Warning
+
+The stryker-karma-runner is available from stryker v0.4.0 onward.
+
+## Install
+
+Install stryker-karma-runner from your project folder:
+
+```bash
+npm i --save-dev stryker-karma-runner
+```
+
+## Peer dependencies
+
+The `stryker-karma-runner` is a pluggin for `stryker` to enable `karma` as a test runner. As such you should install the correct versions of the dependencies:

```

Comment:
The `stryker-karma-runner` is a plugin for `stryker` to enable `karma` as a test runner. As such, you should make sure you have the correct versions of it's dependencies installed: